index.html

```diff
@@ -39,6 +39,156 @@ <h1>I'm José</br>
+    </section>
+    <section class="wrapper">
+        <article>
+           <img src="images/SnapshootPortfolio02.svg">
+           <div class="leftblock">
+               <h2>Tonic</h2>
+               <div class="ref">
+                   <span class="client">Canopy</span>
+                   <span><i class="fa fa-circle fa-xs"></i></span>
+                   <span>Back End Dev</span>
+                   <span><i class="fa fa-circle"></i></span>
+                   <span class="year">2015</span>
+               </div>
+               <p class="primarytext">A daily selection of privately personalized reads; no accounts or sign-ups required.</p> 
+           </div>
+           <ul class="tags">
+               <li class="tag">html</button>
+               <li class="tag">css</button>
+               <li class="tag">javascript</button>
+           </ul>
+           <div class="ref">
+           <button>See project</button>
+           </div>
+        </article>
+   </section>
+   <section class="wrapper">
+         <article>
+            <img src="images/SnapshootPortfolio03.svg">
+            <div class="leftblock">
+                <h2>Tonic</h2>
+                <div class="ref">
+                    <span class="client">Canopy</span>
+                    <span><i class="fa fa-circle fa-xs"></i></span>
+                    <span>Back End Dev</span>
+                    <span><i class="fa fa-circle"></i></span>
+                    <span class="year">2015</span>
+                </div>
+                <p class="primarytext">A daily selection of privately personalized reads; no accounts or sign-ups required.</p> 
+            </div>
+            <ul class="tags">
+                <li class="tag">html</button>
+                <li class="tag">css</button>
+                <li class="tag">javascript</button>
+            </ul>
+            <div class="ref">
+            <button>See project</button>
+            </div>
+         </article>
+    </section>
+    <section class="wrapper">
+        <article>
+           <img src="images/SnapshootPortfolio04.svg">
+           <div class="leftblock">
+               <h2>Tonic</h2>
+               <div class="ref">
+                   <span class="client">Canopy</span>
+                   <span><i class="fa fa-circle fa-xs"></i></span>
+                   <span>Back End Dev</span>
+                   <span><i class="fa fa-circle"></i></span>
+                   <span class="year">2015</span>
+               </div>
+               <p class="primarytext">A daily selection of privately personalized reads; no accounts or sign-ups required.</p> 
+           </div>
+           <ul class="tags">
+               <li class="tag">html</button>
+               <li class="tag">css</button>
+               <li class="tag">javascript</button>
+           </ul>
+           <div class="ref">
+           <button>See project</button>
+           </div>
+        </article>
+   </section>
+    <section class="myself">
```

Comment:
The about myself section looks pretty nice, but please make sure to stick with the Figma template. Because it's one of the basic requirenments.

Here it's required to remove the space from the left and right side of the about section, and also the top-right corner of the section must be rounded.

![image](https://user-images.githubusercontent.com/34779928/192641040-b87f5c85-f101-4a93-98a3-b1d940c6191f.png)